src/apicompat/baselines/ApiCompatBaseline.netstandard2.0.txt

```diff
@@ -1,13 +1,8 @@
-Compat issues with assembly System.Resources.ResourceManager:
-CannotRemoveAttribute : Attribute 'System.ComponentModel.EditorBrowsableAttribute' exists on 'System.Resources.MissingManifestResourceException' in the contract but not the implementation.
-CannotRemoveAttribute : Attribute 'System.ComponentModel.EditorBrowsableAttribute' exists on 'System.Resources.NeutralResourcesLanguageAttribute' in the contract but not the implementation.
-CannotRemoveAttribute : Attribute 'System.ComponentModel.EditorBrowsableAttribute' exists on 'System.Resources.ResourceManager' in the contract but not the implementation.
-CannotRemoveAttribute : Attribute 'System.ComponentModel.EditorBrowsableAttribute' exists on 'System.Resources.SatelliteContractVersionAttribute' in the contract but not the implementation.
-Compat issues with assembly System.Runtime.InteropServices:
-CannotRemoveAttribute : Attribute 'System.ObsoleteAttribute' exists on 'System.Runtime.InteropServices.CustomQueryInterfaceMode' in the contract but not the implementation.
-CannotRemoveAttribute : Attribute 'System.ObsoleteAttribute' exists on 'System.Runtime.InteropServices.CustomQueryInterfaceResult' in the contract but not the implementation.
-CannotRemoveAttribute : Attribute 'System.ObsoleteAttribute' exists on 'System.Runtime.InteropServices.ICustomQueryInterface' in the contract but not the implementation.
-CannotRemoveAttribute : Attribute 'System.ComponentModel.EditorBrowsableAttribute' exists on 'System.Runtime.InteropServices.ComTypes.IStream' in the contract but not the implementation.
-Compat issues with assembly System.Xml.ReaderWriter:
-CannotRemoveAttribute : Attribute 'System.ComponentModel.EditorBrowsableAttribute' exists on 'System.Xml.Schema.XmlSchema' in the contract but not the implementation.
-Total Issues: 9
+Compat issues with assembly System.Memory:
```

Comment:
What are these diffs about?

Reply:
I'm guessing these are real errors, which we weren't seeing before since we just added `System.Memory` to the contract

Reply:
Does that mean we broke the API surface System.Memory previously exposed to `netstandard2.0` consumers?  That'd be a bad issue and should be addressed by adding that surface area to netstandard2.1, or producing a new version of System.Memory that is non breaking.  I don't think that's what is happening, but we need to have an understanding and not guess.  We should be filing issues if we introduce baselines that imply breaking changes.

Reply:
Looks like that's the case, these members (e.g. `TryGetMemoryManager`) exist in the Ref in CoreFx but aren't defined in the Standard (https://github.com/dotnet/corefx/blob/1f9b84a0804e868c7e0f37a3c10fbaf7c735ae14/src/System.Memory/ref/System.Memory.cs#L471). @terrajobst what's the best way for us to add these in Standard?

Reply:
Please open an issue and make sure these are tracked for ns2.1.